### PR TITLE
Add MACB aggregator to MFT

### PR DESCRIPTION
**Pull request by cecinestpasunepipe**
(DIS-2755)

### CODE REVIEW

File: dissect/target/plugins/filesystem/ntfs/mft.py [hunk 1]
```diff
@@ -90,6 +90,21 @@
     ],
 )
 
+FilesystemMCABRecord = TargetRecordDescriptor(
+    "filesystem/ntfs/mft/mcab",
```
Comment (Schamper) [inline dissect/target/plugins/filesystem/ntfs/mft.py:94]:
```suggestion
    "filesystem/ntfs/mft/macb",
```
And everywhere else.

**Review by Schamper — CHANGES_REQUESTED** [review]

File: dissect/target/plugins/filesystem/ntfs/mft.py (hunk 1 above)
Comment (Schamper) [inline dissect/target/plugins/filesystem/ntfs/mft.py:94]:
```suggestion
    "filesystem/ntfs/mft/macb",
```

**Review by Schamper — CHANGES_REQUESTED** [review]

File: tests/plugins/filesystem/ntfs/test_mft.py [hunk 2]
```diff
@@ -170,6 +171,22 @@ def test_mft_plugin_entries(target_win, compact):
     assert len(mft_data) == check_output_amount(76, compact)
 
 
+def test_mft_plugin_mcab(target_win: Target) -> None:
```
Comment (Schamper) [inline tests/plugins/filesystem/ntfs/test_mft.py:174]:
```suggestion
def test_mft_plugin_macb(target_win: Target) -> None:
```

**Review by Schamper — CHANGES_REQUESTED** [review]

File: tests/plugins/filesystem/ntfs/test_mft.py [hunk 3]
```diff
@@ -170,6 +177,40 @@ def test_mft_plugin_entries(target_win, compact):
...
+        tss.add(ts)
+        return ts
+
+    for i in range(0, 100):
+        tss = set()
+        records = []
+        for ts_type in ["M", "A", "C", "B"]:
+            records.append(FilesystemStdRecord(path="a.txt", ts=makets(tss), ts_type=ts_type))
+            records.append(FilesystemFilenameRecord(path="a.txt", ts=makets(tss), ts_type=ts_type, ads=False))
+            records.append(FilesystemFilenameRecord(path="a.txt", ts=makets(tss), ts_type=ts_type, ads=True))
```
Comment (Poeloe) [inline tests/plugins/filesystem/ntfs/test_mft.py:210]:
```suggestion
    def make_ts(tss: set) -> int:
        ts = randrange(1, 10)
        tss.add(ts)
        return ts

    for _ in range(0, 100):
        tss = set()
        records = []
        for ts_type in ["M", "A", "C", "B"]:
            records.append(FilesystemStdRecord(path="a.txt", ts=make_ts(tss), ts_type=ts_type))
            records.append(FilesystemFilenameRecord(path="a.txt", ts=make_ts(tss), ts_type=ts_type, ads=False))
            records.append(FilesystemFilenameRecord(path="a.txt", ts=make_ts(tss), ts_type=ts_type, ads=True))
```

**Review by Poeloe — CHANGES_REQUESTED** [review]

**Review by Poeloe — APPROVED** [review]
LGTM!

**Pull request merged**